tests/test_flow_signal.py

```diff
@@ -72,3 +98,21 @@ class IgnorableSignalFlow(Flow):
     signal_task = flow.Signal(ignorable_test_signal, IgnorableReceiver) \
         .Next(this.end)
     end = flow.End()
+
+
+urlpatterns = [
+    url(r'^signal/', include([
```

Comment:
I think there is a much simpler way exist to test the "issue" with recurcive signals. No urls should be involved.